flow/util/genMetrics.py

```diff
@@ -18,7 +18,7 @@
 import re
 
 # make sure the working dir is flow/
-os.chdir(os.path.join(os.path.dirname(os.path.abspath(__file__)) , '..'))
+# os.chdir(os.path.join(os.path.dirname(os.path.abspath(__file__)) , '..'))
```

Comment:
Why remove?

Reply:
Changes the current directory to ../<script_directory>. 
This genMectrics.py has few problems.

As you see in the below code, it only finds the flow_variant in the current directory (cwd directory). 
Therefore, as the above code changes the current directory, the flow variants may not be available anymore.
https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/8d1989f94a95cfa39bd77c3b538a285689654215/flow/util/genMetrics.py#L908-L927

I should update the root_dir and cwd booth the variables to fix this problem.